### PR TITLE
[fix bug 1170622] Replace jquery templates with handlebars.

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

**Comment by akatsoulas** [comment]
@comzeradd, @johngian r?

**Comment by johngian** [comment]
I suggest we rename ``handlebars_helpers.js`` to ``template_helpers.js``.

File: remo/voting/templates/includes/voting_templates.html [hunk 1]
```diff
@@ -1,167 +1,193 @@
...
+                 type="text">
+        {% endraw %}
       </div>
       <div class="required-field"></div>
-      <div class="formset" data-prefix="${ prefix }-${ count }_radio_choices">
-        <input name="${ prefix }-${ count }_radio_choices-0-id"
-               id="id_${ prefix }-${ count }_radio_choices-0-id"
-               type="hidden">
+      <div class="formset" data-prefix="{{ prefix }}-{{ count }}_radio_choices">
+        {% raw %}<input name="{{ prefix }}-{{ count }}_radio_choices-0-id"
```
Comment (johngian) [inline remo/voting/templates/includes/voting_templates.html]: It looks like ``{% raw %}`` should be in the line above.

File: remo/voting/templates/includes/voting_templates.html [hunk 2]
```diff
@@ -1,167 +1,193 @@
...
+               type="hidden">
+      {% endraw %}
     </div>
   </div>
 </script>
 
-<script id="range-poll-tmpl" type="text/x-jquery-tmpl">
-  <input id="id_${ prefix }-${ count }-id" type="hidden" name="${ prefix }-${ count }-id">
+<script id="range-poll-tmpl" type="text/x-handlebars-template">
+{% raw %}<input id="id_{{ prefix }}-{{ count }}-id" type="hidden"
```
Comment (johngian) [inline remo/voting/templates/includes/voting_templates.html]: Please use a new line for this ``<input>`` element.

File: remo/voting/templates/includes/voting_templates.html [hunk 3]
```diff
@@ -1,167 +1,193 @@
...
+                 type="text">
+        {% endraw %}
       </div>
       <div class="required-field"></div>
-      <div class="formset" data-prefix="${ prefix }-${ count }_range_choices">
-        <input name="${ prefix }-${ count }_range_choices-0-id"
-               id="id_${ prefix }-${ count }_range_choices-0-id"
-               type="hidden">
+      <div class="formset" data-prefix="{{ prefix }}-{{ count }}_range_choices">
+        {% raw %}<input name="{{ prefix }}-{{ count }}_range_choices-0-id"
```
Comment (johngian) [inline remo/voting/templates/includes/voting_templates.html]: Same here.

File: remo/voting/templates/includes/voting_templates.html [hunk 4]
```diff
@@ -1,167 +1,193 @@
...
         <div class="copyblock nominee">
           <div class="select-nominee-group">
             <div title="Select Nominee"
-                 data-tooltip
-                 class="has-tip tip-left select-nominee">
-              <select id="id_${ prefix }-${ count }_range_choices-0-nominee"
-                      name="${ prefix }-${ count }_range_choices-0-nominee">
+                  data-tooltip
+                  class="has-tip tip-left select-nominee">
+              {% raw %}<select id="id_{{ prefix }}-{{ count }}_range_choices-0-nominee"
```
Comment (johngian) [inline remo/voting/templates/includes/voting_templates.html]: Same here.

File: remo/voting/templates/includes/voting_templates.html [hunk 5]
```diff
@@ -1,167 +1,193 @@
...
+                               name="{{ prefix }}-{{ count }}_range_choices-0-nominee">{% endraw %}
                 <option selected="selected" value="">---------</option>
-                {% for nominee in nominee_list %}
+                {% for nominee in nominee_list %}}
```
Comment (johngian) [inline remo/voting/templates/includes/voting_templates.html]: Is this a typo?

File: remo/voting/templates/includes/voting_templates.html [hunk 6]
```diff
@@ -1,167 +1,193 @@
...
                   <option value="{{ nominee.id }}">{{ nominee }}</option>
-                {% endfor %}
+                {% endfor %}}
```
Comment (johngian) [inline remo/voting/templates/includes/voting_templates.html]: Same here.

File: remo/voting/templates/includes/voting_templates.html (hunk 4 above)
Comment (akatsoulas) [inline remo/voting/templates/includes/voting_templates.html]: Although I don't have strong feelings about it, I believe that in some cases is easier to read it inline.

**Comment by akatsoulas** [comment]
@johngian pr updated. Maybe you could do another pass?

File: remo/voting/templates/includes/voting_templates.html (hunk 4 above)
Comment (johngian) [inline remo/voting/templates/includes/voting_templates.html]: While reviewing this, it seemed to me very unreadable. I don't have any strong opinions about this though.

**Comment by johngian** [comment]
You also need to fix the profile listing ``profiles_people.html``. At the moment `/people` is  broken.

**Comment by akatsoulas** [comment]
@johngian pr updated. Final pass?

**Comment by johngian** [comment]
Code looks good.
Tested the following locally:
* `/events` list
* `/people` list
* `/voting` formset manipulation

r+ :rabbit:

**Pull request merged**